screed/screedRecord.py

```diff
@@ -129,6 +112,18 @@ def __ne__(self, given):
         except AttributeError:
             raise TypeError("Cannot compare to given type: %s" % type(given))
 
+    def __lt__(self):
+        return NotImplementedError
```

Comment:
Ah..Perhaps? The python docs were confusing--seemed to suggest that returning the `NotImplementedError` object was the thing to do. I did a quick search for a `NotImplemented` exception and didn't see anything.